tests/Tutor.Web.Tests/Integration/BaseWebIntegrationTest.cs

```diff
@@ -31,6 +33,15 @@ protected KcMasteryController SetupKcmController(IServiceScope scope, string use
             };
         }
 
+        protected InstructorController SetupInstructorController(IServiceScope scope, string userAndInstructorId)
```

Comment:
Leave this for now, but we should really restructure the BaseWebIntegrationTest as it currently exhibits Refused Bequest https://refactoring.guru/smells/refused-bequest